CodeGenBringUpTests/Rotate.cs forgets third constructor parameter

Line 291
https://github.com/dotnet/coreclr/blob/master/tests/src/JIT/CodeGenBringUpTests/Rotate.cs#L291

Obviously Test constructor takes third parameter. Error jenkins log from last *successful* run:

> 22:20:57 Rotate.cs(291,25): error CS7036: There is no argument given that corresponds to the required formal parameter 'k' of 'Test.Test(ulong, uint, ushort)' [d:\j\workspace\dotnet_coreclr\debug_windows_nt\tests\src\JIT\CodeGenBringUpTests\Rotate.csproj]

http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/debug_windows_nt/219/consoleFull

@erozenfeld

--- a/tests/src/JIT/CodeGenBringUpTests/Rotate.cs
+++ b/tests/src/JIT/CodeGenBringUpTests/Rotate.cs
@@ -288,7 +288,7 @@ public class Test
             return Fail;
         }
 
-        Test test = new Test(0x123456789abcdef, 0x12345678);
+        Test test = new Test(0x123456789abcdef, 0x12345678, 0x1234);
 
         if (test.rol64field(11) != 0x1a2b3c4d5e6f7809)
         {

